### DISCUSSION TITLE
anyone maintaining this? i see huge potential for wider adoption but it stopped updating 2 years ago.

### DISCUSSION THREAD

**Discussion by sprappcom** [open2b/scriggo#949] [Q&A]
as titled

**Comment by zapateo** [comment]
Hello @sprappcom, we apologize for the delay in our response.

At the moment, we do not plan to extend Scriggo with new features, but we are still working on updating it to support the new versions of the Go compiler.

As you can see from the [Releases section](https://github.com/open2b/scriggo/releases), just today (October 22, 2024), we released versions v0.57.0 and v0.57.1, which support Go compiler versions 1.22 and 1.23.

You can also find this information on the [Scriggo website, News section](https://scriggo.com/news).

Thank you for your support and for the attention you have given to Scriggo. Feel free to start new discussions or open issues if you encounter any problems.

Have a great day! 🙂

**Comment by zapateo** [comment]
For anyone reading this discussion in 2026 or later, we've released a [new version of Scriggo, the v0.61.0](https://github.com/open2b/scriggo/discussions/1015). 🎉
